packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -108,6 +108,24 @@ export interface BedrockInvokeModelProps extends sfn.TaskStateBaseProps {
    */
   readonly output?: BedrockInvokeModelOutputProps;
 
+  /**
+   * The destination location where the API response is written.
+   *
+   * This field can be used to specify s3 URI in the form of token
+   *
+   * @default - The API response body is returned in the result.
+   */
+  readonly s3InputUri?: string;
```

Comment:
Can we also update the [Readme](https://github.com/aws/aws-cdk/blob/f3bc16c7b9f1d58318a451d8640a0f8f07f4ffcb/packages/aws-cdk-lib/aws-stepfunctions-tasks/README.md?plain=1#L402-L422)